From 23ea5504200ff5f7e40bd264280a78db09c1bc5b Mon Sep 17 00:00:00 2001
From: Johan Klokkhammer Helsing <johan.helsing@qt.io>
Date: Fri, 8 Nov 2019 13:58:04 +0100
Subject: [PATCH] WIP: Fix compilation of linuxdmabuf compositor plugin

WIP, because I'm not sure if this is the way to go, but it fixes
compilation on my machine.

Fixes: QTBUG-79709
Change-Id: I3190ef56e0e162636efea440dff7e760cf11fcd0
---
 .../compositor/linux-dmabuf-unstable-v1/linuxdmabuf.h                    | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/hardwareintegration/compositor/linux-dmabuf-unstable-v1/linuxdmabuf.h b/src/hardwareintegration/compositor/linux-dmabuf-unstable-v1/linuxdmabuf.h
index 2abc2ce6b..d1f4a3039 100644
--- a/src/hardwareintegration/compositor/linux-dmabuf-unstable-v1/linuxdmabuf.h
+++ b/src/hardwareintegration/compositor/linux-dmabuf-unstable-v1/linuxdmabuf.h
@@ -53,6 +53,7 @@
 
 #include <EGL/egl.h>
 #include <EGL/eglext.h>
+#include <EGL/eglmesaext.h>
 
 // compatibility with libdrm <= 2.4.74
 #ifndef DRM_FORMAT_RESERVED
-- 
2.16.3

